content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/cordapp-packaging.md

```diff
@@ -1,30 +1,30 @@
 ---
-date: '2021-09-21'
+date: '2021-11-14'
 title: "CorDapp Packaging"
-draft: true
 menu:
   corda-5-alpha:
-    parent: corda-5-alpha-tutorials-dev
-    identifier: corda-5-alpha-tutorial-dev-one
-    weight: 1000
+    parent: corda-5-alpha-tutorials-deploy
+    identifier: corda-5-alpha-tutorial-deploy-cordapp-packaging
+    weight: 2000
 section_menu: corda-5-alpha
 ---
 
-This section describes how to package your CorDapp as CPKs, CPBs, and CPIs. You can read more about the CorDapp packaging format in the [Key concepts](../../introduction/key-concepts.html#packaging) section.
+This section describes how to package your CorDapp as CPKs, CPBs, and CPIs.
 
 ## Before You Start
 
-You must install [Corda CLI](../corda-cli/overview.html).
+You will need a working copy of `corda-cli` with the MGM and package plugins installed. 
+<!-- See the [Corda CLI Plugin Host](https://github.com/corda/corda-cli-plugin-host#setupbuild). -->
 
-<!--## Configure the plugin in gradle???
+## Configuring the Gradle Plugin
 
-This describes how to convert an existing CorDapp project to the new Gradle plugin.
+To convert an existing CorDapp project to the new Gradle plugin:
 
 1. Add a new version number to `gradle.properties`:
     ```groovy
     cordaGradlePluginsVersion2=7.0.0-SNAPSHOT
     ```
-1. Add this repository to pluginManagement/repositories in `settings.gradle`:
+2. Add this repository to pluginManagement/repositories in `settings.gradle`:
     ```groovy
     maven {
         url "${artifactoryContextUrl}/corda-dev"
```

Comment:
Should we delete this and suggest cloning the CSDE template repo as a starting point?

Reply:
The CSDE template is already configured but I guess it is handy to have standalone instructions for when we need them.

Reply:
The version number is now "7.0.1-SNAPSHOT" in the git repo.

Reply:
The CSDE is not included in the Alpha release